src/lib/util/logger.js

```diff
@@ -6,20 +6,38 @@ const auditLogEnabled = process.env.AUDIT_LOGGING !== 'false' && process.env.AUD
 function auditLog (success, request, info, parent, args, msg) {
   if (auditLogEnabled) {
     auditLogger.info({
-      msg: msg || '',
-      requestId: request ? request.id : '',
-      operationType: info.operation.operation,
-      fieldName: info.fieldname,
-      parentTypeName: info.parentType.name,
-      path: buildPath(info.path),
-      success: success,
-      parent: parent,
-      arguments: args,
-      dataSourceType: info.dataSourceType || ''
+      audit: {
+        msg: msg || '',
+        requestId: request ? request.id : '',
+        operationType: info.operation.operation,
+        fieldName: info.fieldname,
+        parentTypeName: info.parentType.name,
+        path: buildPath(info.path),
+        success: success,
+        parent: parent,
+        arguments: args,
+        dataSourceType: info.dataSourceType || '',
+        clientInfo: getClientInfoFromHeaders(request)
+      }
     })
   }
 }
 
+function getClientInfoFromHeaders (request) {
+  if (request && request.headers && request.headers['client-info']) {
```

Comment:
No strong opinion, but this seems to be too generic for me. I will prefer key to be more unique like data-sync-client-info or something like this